tools/doc-generator/doc-generator.go

```diff
@@ -15,6 +15,34 @@ import (
 	_ "kubevirt.io/kubevirt/pkg/virt-controller/watch"
 )
 
+const (
+	docFileName = "metrics.md"
+
+	headerMarkdown = `# KubeVirt metrics
+
+This document aims to help users that are not familiar with all metrics exposed by different KubeVirt components. 
+All metrics documented here are auto-generated by the utility tool` + "`tools/doc-generator`" + ` and reflects exactly what is being exposed.
```

Comment:
nit: 
```suggestion
All metrics documented here are auto-generated by the utility tool`tools/doc-generator` and reflect exactly what is being exposed.
```

Reply:
I think this would be incorrect. I'm using backticks to create a multiline string and that would close the multiline string.
Unfortunately, I couldn't find a way to escape the backtick, so that was the way I found to use them inside the string.

Reply:
I was just wanting to change `reflects` to `reflect`.